src/words_hw4.py

```diff
@@ -0,0 +1,34 @@
+"""Во входной строке записан текст. Словом считается
+ последовательность непробельных символов идущих подряд,
+ слова разделены одним или большим числом пробелов
+  или символами конца строки. Определите, сколько различных
+  слов содержится в этом тексте.
+"""
+# 1) если надо вывести количество слов которые не повторяются
```

Comment:
it seems .split would be enough. 
this task has one-line solution with set comprehension.